examples/step-19/step-19.cc

```diff
@@ -760,9 +713,9 @@ namespace Step19
                 particle->set_properties(make_array_view(new_velocity));
 
                 // With the new velocity, we can then also update the location
-                // of the particle and set tell the particle about it.
+                // of the particle and tell the particle about it.
                 const Point<dim> new_location =
-                  particle->get_location() + dt * old_velocity;
+                  particle->get_location() + dt * new_velocity;
```

Comment:
If I understand the formula and the comment above correctly, it should be the new velocity here, shouldn't it?

Reply:
That's the difference between the explicit Euler and some semi-implicit scheme. For the purposes of this program, it probably doesn't matter very much. You're right that what you have here matches the formula above -- what did I write in the introduction? I think I *wanted* to use the explicit Euler scheme.

Reply:
I will let you go over it after this PR is merged. I note that the introduction states this one here:
https://github.com/dealii/dealii/blob/f0919993dd8325c993eea8af248e233c1a11bd59/examples/step-19/doc/intro.dox#L180-L186
So this corresponds to some operator splitting (Gauss-Seidel) type approach.
Later, the results section goes on to mention the forward Euler case
https://github.com/dealii/dealii/blob/f0919993dd8325c993eea8af248e233c1a11bd59/examples/step-19/doc/results.dox#L265-L272
which is incorrect because the time `n` refers to a backward Euler case. But this is not really a backward Euler in the sense of differential algebraic equations because the electrical potential does not refer to the new values, but rather the splitting. I will open an issue for this.